### PR TITLE
rec: Modify per-thread cpu usage stats to be Prometheus-friendly

### PR DESCRIPTION
**Pull request by omoerbeek**
Plus fix a few small issues wrt HELP texts.

Example output:
```
# HELP pdns_recursor_cpu_msec Number of milliseconds spent in thread n
# TYPE pdns_recursor_cpu_msec counter
pdns_recursor_cpu_msec{thread=0} 10
pdns_recursor_cpu_msec{thread=1} 0
pdns_recursor_cpu_msec{thread=2} 0
```

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [X] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Comment by omoerbeek** [comment]
Rebased to fix conflict

File: pdns/recursordist/rec_metrics.hh [hunk 1]
```diff
@@ -83,6 +84,9 @@ public:
     case PrometheusMetricType::histogram:
       return "histogram";
       break;
+    case PrometheusMetricType::multicounter:
+      return "multicounter";
```
Comment (rgacogne) [inline pdns/recursordist/rec_metrics.hh:88]: I wonder if returning "counter" here would not make the code in `prometheusMetrics()` simpler, but that does not matter much.

**Review by rgacogne — APPROVED** [review]

File: pdns/recursordist/rec_metrics.hh (hunk 1 above)
Comment (omoerbeek) [reply pdns/recursordist/rec_metrics.hh:88]: I'll take a look at that when finishing #10605 (which depends on this PR)

**Comment by KVInventoR** [comment]
Hi @omoerbeek, could you please check: https://github.com/PowerDNS/pdns/issues/10735

**Pull request merged**